wdk-ddi-src/content/wdm/nc-wdm-po_fx_directed_power_up_callback.md

```diff
@@ -80,6 +80,8 @@ WDM drivers that register with PoFx for runtime idle power management support ne
 
 Register your implementation of this callback function by setting the appropriate member of the [**PO_FX_DEVICE_V3**](ns-wdm-po_fx_device_v3.md) structure and then calling [**PoFxRegisterDevice**](./nf-wdm-pofxregisterdevice.md).
 
+This callback is invoked when the system exists DRIPS, not Modern Standby. A PEP driver determines the timing when the system exists DRIPS.

```

Comment:
Recommend: "This callback is invoked when the system determines the device needs to power up after having previously being powered down through [DirectedPowerDown](https://docs.microsoft.com/en-us/windows-hardware/drivers/ddi/wdm/nc-wdm-po_fx_directed_power_down_callback)."